sdk/api/metalava.txt

```diff
@@ -570,6 +600,9 @@ package com.mapbox.maps.renderer.widget {
     enum_constant public static final com.mapbox.maps.renderer.widget.WidgetPosition.Vertical TOP;
   }
 
+  public final class WidgetPositionKt {
```

Comment:
can we avoid having this in api file?

Reply:
I guess it's because of `@JvmSynthetic fun WidgetPosition`. I wonder if marking the whole `WidgetPosition.kt` file with `@file:JvmSynthetic` would hide it.

Reply:
I didn't find a way to hide this empty class, as we have a DSL function inside the `WidgetPosition` class that was marked as `@JvmSynthetic`, the function itself is hidden from Java, but produced this empty class anyway, any ideas? otherwise I guess we can leave it here.

Reply:
> I wonder if marking the whole WidgetPosition.kt file with @file:JvmSynthetic would hide it.

I actually tried this, however then it produced:
```
 @kotlin.jvm.JvmSynthetic public final class WidgetPositionKt {
   }
```
🙈